R/imputation.R

```diff
@@ -26,7 +26,11 @@ setMethod("impute", "MSnSet",
               ## imputaton methods
               ##
               if (method == "knn") {
-                  .eset <- impute.knn(exprs(object), ...)$data
+                  imp_res <- impute.knn(exprs(object), ...)
+                  .eset <- imp_res$data
+                  if(! is.null(imp_res$rng.state)){
```

Comment:
Just a minor coding style correction (spaces at the wrong place):
```suggestion
                  if(!is.null(imp_res$rng.state)) {
```